gremlin/values.yaml

```diff
@@ -24,3 +24,4 @@ gremlin:
     secretName: gremlin-team-cert
     cert: file:///var/lib/gremlin/cert/gremlin.cert
     key: file:///var/lib/gremlin/cert/gremlin.key
+  clientTags: ""
```

Comment:
Any thoughts on shortening this down to `tags`? Since it's namespace'd under `gremlin` I don't think there's any ambiguity as to what tags these are.

Reply:
Sure thing.